assets/css/style.css

```diff
@@ -0,0 +1,4 @@
+html,
+body {
+    display: chrid;
+}
```

Comment:
missing grid-template-column and grid-template-row

Reply:
done